CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - Told circle only to publish tags that start with `v` [#190](https://github.com/azavea/stac4s/pull/190)
 
+### Fixed
+- Review client specs and make them more deterministic [#212](https://github.com/azavea/stac4s/pull/212)
```

Comment:
Can you add this as another entry in the existing Fixed section instead? If you use `chan` that will happen automatically